crates/host/src/wasmbus/mod.rs

```diff
@@ -318,79 +336,39 @@ struct Handler {
     origin: WasmCloudEntity,
     // package -> target -> entity
     links: Arc<RwLock<HashMap<String, HashMap<String, WasmCloudEntity>>>>,
-    /// The current link name to use for interface targets, overridable in actor code via set_target()
+    /// The current link name to use for interface targets, overridable in actor code via set_link_name()
     interface_link_name: Arc<RwLock<String>>,
-    // link name -> package -> interface -> target
+    // link name -> package -> interface -> component ID
```

Comment:
I think this should still just be `target`, but I'm fine with brainstorming other ways to specify it. Reason being is that the target here is not always a component ID, it could be a routing group or an actor nkey etc etc